models/surveys.go

```diff
@@ -359,17 +369,16 @@ func (api *API) insertClassifierTypes(classifierTypes []string, typeSelectorPK i
 }
 
 // Insert a classifier type selector into the database given survey primary key using transaction tx, return the PK and UUID
-func (api *API) insertClassifierTypeSelector(name string, surveyPK int, tx *sql.Tx) (int, uuid.UUID, error) {
+func (api *API) insertClassifierTypeSelector(name string, surveyPK int, classifierTypeSelectorID uuid.UUID, tx *sql.Tx) (int, error) {
```

Comment:
Why are we passing one in now instead of creating one?

Reply:
Put it back to how it was